standards/naming.md

```diff
@@ -29,7 +29,7 @@ When you consider the names to use for your path, endpoints and other components
 - Names **MUST** use the same term for the same concept used elsewhere, including concepts shared across APIs. Refer to specific components below.
 - Names **MUST** be different for different concepts.
 - Overly general names that are ambiguous within the context of the API and the larger ecosystem of APIs **MUST** be avoided (i.e. "identity", "notification-service", etc). They can lead to a misunderstanding of API concepts. Rather, choose specific names that accurately describe the API concept. - This is particularly important for names that define first-order API elements, such as resources. There is no definitive list of names to avoid, as every name must be evaluated in the context of other names.
-- Names that may conflict with keywords in common programming languages **SHOULD NOT** be used. 
+- Names that may conflict with keywords in common programming languages **SHOULD NOT** be used. <a name="sps-no-keyword-conflicts" href="#sps-no-keyword-conflicts"><i class="fa fa-check-circle" title="#sps-no-keyword-conflicts"></i></a>
```

Comment:
Are there any more opportunities to add these page anchors across the document? We added ~20 new rules

Reply:
There are a lot of rules without anchors in this doc. Let's get them added